src/components/ScreenSurface.tsx

```diff
@@ -35,6 +35,7 @@ export function ScreenSurface({
 const defaultStyles = createStyles('ScreenSurface', (theme) => ({
   surfaceView: {
     flex: 1,
+    height: '100%',
```

Comment:
After the upgrade a react native paper surface must have a height set to flex properly.